mypy.ini

```
@@ -82,6 +82,72 @@ ignore_errors = True
 [mypy-manim.utils.*]
 ignore_errors = True
 
+[mypy-manim.utils.bezier.*]
+ignore_errors = False
+
+[mypy-manim.utils.commands.*]
+ignore_errors = False
+
+[mypy-manim.utils.config_ops.*]
+ignore_errors = False
+
+[mypy-manim.utils.color.*]
+ignore_errors = False
+
+[mypy-manim.utils.debug.*]
+ignore_errors = False
+
+[mypy-manim.utils.deprecation.*]
+ignore_errors = False
+
+[mypy-manim.utils.docbuild.*]
+ignore_errors = True
+
+[mypy-manim.utils.family_ops.*]
+ignore_errors = False
+
+[mypy-manim.utils.file_ops.*]
+ignore_errors = False
+
+[mypy-manim.utils.ipython_magic.*]
+ignore_errors = False
+
+[mypy-manim.utils.module_ops.*]
+ignore_errors = False
+
+[mypy-manim.utils.opengl.*]
+ignore_errors = False
+
+[mypy-manim.utils.parameter_parsing.*]
+ignore_errors = False
+
+[mypy-manim.utils.paths.*]
+ignore_errors = False
+
+[mypy-manim.utils.rate_functions.*]
+ignore_errors = False
+
+[mypy-manim.utils.sounds.*]
+ignore_errors = False
+
+[mypy-manim.utils.space_ops.*]
+ignore_errors = False
+
+[mypy-manim.utils.simple_functions.*]
+ignore_errors = False
+
+[mypy-manim.utils.testing.*]
+ignore_errors = False
+
+[mypy-manim.utils.tex_file_writing.*]
+ignore_errors = False
+
+[mypy-manim.utils.tex_templates.*]
+ignore_errors = False
+
+[mypy-manim.utils.unit.*]
+ignore_errors = False
+
 [mypy-manim.utils.iterables]
 ignore_errors = False
 warn_return_any = False
```

If you typed all of `utils`, then you can remove all of this. In this way, by default, no errors will be ignored.
```suggestion
[mypy-manim.utils.iterables]
warn_return_any = False
```